api/routes/users.js

```diff
@@ -59,6 +60,21 @@ module.exports = async (req, res) => {
       query = query.whereBetween('last_edit', [subMonths(Date.now(), 6), Date.now()])
     }
 
+    if (sortType.length > 0) {
+      if (sortType === 'Most recent') {
+        query = query.orderBy('last_edit', 'desc')
```

Comment:
`query = query.orderBy('last_edit', 'asc')` is repeated multiple times. I wonder if there is a way to simplify this a bit.